pages/events/[slug].tsx

```diff
@@ -30,7 +30,7 @@ const Page: NextPage<PageProps> = ({ event, project, owner, events }) => {
   return (
     <Layout
       crumbs={[
-        { path: "/portal-dobrovolnika", label: "Portál dobrovolníka" },
+        { path: "/dashboard", label: "Portál dobrovolníka" },
```

Comment:
Ideálně ale `Route.volunteerPortal`? (Snažíme se zařídit, aby ty routy byly na jednom místě a daly se snáz měnit.) A případně můžeš `Route.volunteerPortal` přejmenovat na `Route.dashboard`, ať je to konzistentní.